utility/src/main/java/org/oppia/android/util/parser/GlideImageLoader.kt

```diff
@@ -36,7 +36,7 @@ class GlideImageLoader @Inject constructor(
     Glide.with(context)
       .asBitmap()
       .load(model)
-      .transform(*transformations.toGlideTransformations())
+      .dontTransform()
```

Comment:
Could you please be more descriptive on this why we don't need transformation?

Reply:
@anandwana001 since the images have already been pre-scaled

Reply:
Could you check does this change affecting any other part of the app or not?

Reply:
@anandwana001 it doesn't affect anything

Reply:
Could you confirm the story screen, go to lesson click on topic, it will open a story activity with a list of cards, and check if any card is blur or not!!

Reply:
@anandwana001 I checked it on an emulator (pixel 5 API 29) and realme 5 (API 30) I found no new issues created.